tools/c-parser/README.md

```diff
@@ -87,7 +89,7 @@ enclosed in square brackets can be provided. These are:
 An example that uses all options:
 
 ```isabelle
-install_C_file memsafe c_types c_defs "file.c"
+install_C_file memsafe c_types c_defs no_modifies "file.c"
                [machinety=int, ghostty=nat, roots=[f, g, h]]
```

Comment:
worth explaining *why* we don't bother with modifies theorems during the kernel pass, maybe a pointer to when they do happen?

Reply:
Done.